static/sass/_pattern_cube-image.scss

```diff
@@ -0,0 +1,16 @@
+@mixin ubuntu-p-cube-image {
+  .p-cube-image {
```

Comment:
It would be nice to name this something reuseable. Can it be changed to something reflecting the effect, not the look or where it's used?